x-pack/test_serverless/api_integration/test_suites/common/data_usage/metrics.ts

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import http from 'http';
+
+import { SupertestWithRoleScope } from '@kbn/test-suites-xpack/api_integration/deployment_agnostic/services/role_scoped_supertest';
+import { UsageMetricsRequestBody } from '@kbn/data-usage-plugin/common/rest_types';
+import { FtrProviderContext } from '../../../ftr_provider_context';
+import { createProxyServer } from './proxy_server';
+
+const API_PATH = '/internal/api/data_usage/metrics';
+export default function ({ getService }: FtrProviderContext) {
+  const svlDatastreamsHelpers = getService('svlDatastreamsHelpers');
+  const roleScopedSupertest = getService('roleScopedSupertest');
+  const log = getService('log');
+  let supertestAdminWithCookieCredentials: SupertestWithRoleScope;
+  describe('Metrics', function () {
+    // proxy does not work with MKI
+    this.tags(['skipMKI']);
```

Comment:
these tests wont run in mki due to proxy not working.  will look into seeing if we can get around this.